src/libs/GoogleTagManager/index.ts

```diff
@@ -0,0 +1,35 @@
+import Log from '@libs/Log';
+import type {GoogleTagManagerEvent} from './types';
+import type GoogleTagManagerModule from './types';
+
+/**
+ * The dataLayer is added with a js snippet from Google in web/thirdPartyScripts.js. Set USE_THIRD_PARTY_SCRIPTS to true
+ * in your .env to enable this
+ */
+type WindowWithDataLayer = Window & {
+    dataLayer?: {
+        push: (params: DataLayerPushParams) => void;
+    };
+};
+
+type DataLayerPushParams = {
+    event: GoogleTagManagerEvent;
+    accountID: number;
+};
+
+declare const window: WindowWithDataLayer;
+
+function publishEvent(event: GoogleTagManagerEvent, accountID: number) {
+    if (!window.dataLayer) {
+        return;
```

Comment:
NAB: Should we log when this happens, maybe a warning?

Reply:
Nah it'll be expected for most of us in dev without `USE_THIRD_PARTY_SCRIPTS ` enabled